public/terms.html

```diff
@@ -0,0 +1,419 @@
+          </p>
+
+          <h5>Termination and Suspension.</h5>
+          <p>
+            We may terminate or suspend the Website or any part of the Website,
+            terminate or suspend your use of the Website, block any IP address,
+            or remove any of your User Content at any time without cause without
+            any liability to you.
+          </p>
+
+          <p>
+            Further, we may terminate or suspend your permission to use the
+            Website immediately and without notice upon any violation of these
+            Terms of Service, your failure to pay any fees when due, upon the
+            request of law enforcement or government agencies, for extended
+            periods of inactivity, for unexpected technical issues or problems
+            or for engagement by you in fraudulent or illegal activities. If we
+            terminate your use of the Website for any of these reasons or
+            otherwise for cause, we will not refund any fees you may have paid,
+            whether for access to the Website or for Software (if applicable).
+          </p>
+
+          <p>
+            Upon any termination we may delete your account, passwords and User
+            Content, and we may bar you from further use of the Website. You
+            understand that we may also continue to make your User Content
+            available on the Website even if your use of the Website is
+            terminated or suspended. You agree that we will have no liability to
+            you or any third party for termination of your account or access to
+            the Website.
+          </p>
+
+          <h5>General Terms.</h5>
+          <p>
+            We maintain the Website in the United States of America and the
+            United Kingdom. and you agree that these Terms of Service and any
+            legal action or proceeding relating to the Website shall be governed
+            by the laws of the United States of America and United Kingdom
+            without reference to its choice of law rules. If you attempt to
+            bring any legal proceedings against us you specifically acknowledge
+            that we are free to choose the jurisdiction of our preference as to
+            where such action against us may be held. As you have agreed by
+            using the Website to choose the laws of the United States of America
+            to govern any such proceedings, we will probably choose to defend
+            any such action in England and we can make this decision entirely as
+            it suits us, without regard to where in the world you are located,
+            or from where in the world you visited the Website. You are
+            responsible for complying with the laws of the jurisdiction from
+            which you are accessing the Website and you agree that you will not
+            access or use the information on the Website in violation of such
+            laws.
```

Comment:
I'm not totally convinced with this paragraph tho.

Reply:
I'm not a legal expert so I'm not sure how necessary it is.. I usually err on the side of saying less, especially as legal matters are present so if it's not making an essential point, I'd scratch it.

Reply:
Okay, I'll ask one of my friends who studied Law and see if she can shed a light on it (since I believe such paragraph may be necessary but I'm also not an expert).

Reply:
> I believe the situation is that **if you take the paragraph out and don't state a jurisdiction and a dispute arises, then a court or arbitrator will decide which law has jurisdiction (UK or USA)**. Therefore, I would be inclined to leave it out but I'm not an expert in this,

According to her, it can be removed. We can always amend that later on tho.